app/config/collections2.php

```diff
@@ -360,6 +360,254 @@
         ],
     ],
 
+    'sessions' => [
+        '$collection' => Database::COLLECTIONS,
+        '$id' => 'sessions',
+        'name' => 'Sessions',
+        'attributes' => [
+            [
+                '$id' => 'userId',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'provider',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 1000,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'providerUid',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 2048,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'providerToken',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'secret',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 64, // https://www.tutorialspoint.com/how-long-is-the-sha256-hash-in-mysql
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'expire',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'userAgent',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'ip',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 15,
```

Comment:
We should make this future proof and compatible with IP v6 🤔

Reply:
Yeh, this is not even the most optimized way to store IP in a database, just wanted to avoid any complex conversions. @kodumbeats please let me know what you think.

Reply:
We should set ourselves up to store both IPv4 and IPv6 in one column, so I think a string is okay for now. Looks like the largest string representation of an IPv6 address is **45 characters** ([ref](https://stackoverflow.com/questions/166132/maximum-length-of-the-textual-representation-of-an-ipv6-address))

Since each DB will handle native searching/storing IPs differently, let's do that in a separate PR.